src/pca/pca.jl

```diff
@@ -0,0 +1,102 @@
+using Statistics
+using LinearAlgebra
+
+export PCA
```

Comment:
I am not sure that PCA shouldbe an exported name now that we will move on with MultiVariateStats. I'd suggest to not export it.

Reply:
Forget my comment. If we move on with Multivar I assume the entire `pca.jl` file will get deleted.